linux/workcraft.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=Workcraft
+Comment=A framework for interpreted graph models including petri nets and STGs
+Exec=workcraft
+Icon=workcraft
```

Comment:
Do we need to rename icon.svg into workcraft.svg for this?

Reply:
There shouldn't be any need to rename the icon.

.desktop icon key:
> If the name is an absolute path, the given file will be used. If the name is not an absolute path, the algorithm described in the Icon Theme Specification will be used to locate the icon.

In the AUR package I've used the latter option and copied the icon to `/usr/share/icons/hicolor/scalable/apps/workcraft.svg`. Since the installation path of Workcraft could differ depending on distro, I think this is the best way.